src/video_core/renderer_opengl/renderer_opengl.cpp

```diff
@@ -500,6 +507,17 @@ bool RendererOpenGL::Init() {
 
     return true;
 }
+void RendererOpenGL::FrameLimiter(float average_frame_time,int framelimit)
+{
+    //calculate  difference between average frame time and frame time for the framelimit, and sleep for the difference in time.
+    float frame_time = 1000 / framelimit;
+    if (average_frame_time < frame_time)
+    {
```

Comment:
I believe so. When in doubt, check other occurences in existing code or run clang-format.